### ISSUE TITLE
Can't import scipy

### ISSUE THREAD

**Issue by leynier** [scipy/scipy#10391]
My issue is about the import of scipy

### Reproducing code example:

```
import scipy
```

### Error message:

```
Traceback (most recent call last):
  File "D:/Projects/gait/tests/__init__.py", line 2, in <module>
    import scipy
  File "C:\Users\Leynier\AppData\Local\Programs\Python\Python37-32\lib\site-packages\scipy\__init__.py", line 110, in <module>
    from scipy._lib._version import NumpyVersion as _NumpyVersion
  File "C:\Users\Leynier\AppData\Local\Programs\Python\Python37-32\lib\site-packages\scipy\_lib\__init__.py", line 14, in <module>
    from scipy._lib._testutils import PytestTester
ValueError: source code string cannot contain null bytes

Process finished with exit code 1
```

### Scipy/Numpy/Python version information:

> * Scipy version: `1.3.0`
> * Numpy version: `1.16.4`
> * Python version: `3.7.3`
> * OS: `Windows 10`

**Comment by rgommers** [comment]
This looks like a failed install or incomplete build. Not much more we can say without a build log. 

Not a SciPy issue, so closing. If you still have trouble @leynier please feel free to continue the discussion. We need more details on your install method in that case.

**Comment by leynier** [comment]
I solved the problem by installing Python 3.6 and installing Scipy in that version. Because with version 3.7 the problem continued even if I reinstalled Scipy. The installation method I use is pip.

**Comment by zhengfen** [comment]
I just met the same issue, thanks to leynier, I updated python to 3.8 and problem solved.
